modules/apps/27-interchain-accounts/host/client/cli/tx.go

```diff
@@ -74,7 +77,16 @@ which submits pre-built packet data containing messages to be executed on the ho
 				return err
 			}
 
-			packetDataBytes, err := generatePacketData(cdc, []byte(args[0]), memo)
+			encoding, err := cmd.Flags().GetString(encodingFlag)
+			if err != nil {
+				return err
+			}
+
+			if !slices.Contains([]string{icatypes.EncodingProtobuf, icatypes.EncodingProto3JSON}, encoding) {
```

Comment:
`slices` is still in the exp pkg on go1.19, should we use the internal `collections.Contains()` that we have at the moment, and update all occurrences to use `slices.Contains` when eden is merged? 

I don't feel too strongly about this, I'm also happy to keep this as is 👍

Reply:
Yeap, I can change it.